src/java/org/apache/cassandra/index/sai/utils/TermsIterator.java

```diff
@@ -32,13 +31,8 @@
  * Term enumerations are always ordered by their {@link ByteSource}.
  */
 @NotThreadSafe
-public interface TermsIterator extends Iterator<ByteComparable>, Closeable
+public interface TermsIterator extends Iterator<IndexEntry>, Closeable
 {
-    /**
-     * Get {@link PostingList} for the current term.
-     */
-    PostingList postings();
```

Comment:
Removing this method makes the first line in the class Javadoc a bit obsolete.

Reply:
Apologies. I had meant to update that javadoc and forgot. I've now updated it.